llvm/lib/Target/RISCV/RISCVInstrInfoP.td

```diff
@@ -25,7 +25,7 @@ def SImm8UnsignedAsmOperand : SImmAsmOperand<8, "Unsigned"> {
 }
 
 // A 8-bit signed immediate allowing range [-128, 255]
-// but represented as [-128, 127].
+// but represented as [-128, 255].
```

Comment:
What do you mean by "but" if the two ranges are identical?

Reply:
Looks like I reintroduced the issue Sam commented on here https://github.com/llvm/llvm-project/pull/153913#discussion_r2280631322